tests/utils/openai.py

```diff
@@ -205,7 +206,7 @@ async def get_response() -> ChatCompletion:
             stream=stream,
             stop=stop,
             max_tokens=max_tokens,
-            temperature=0.0,
+            temperature=temperature if temperature is not None else 0.0,
```

Comment:
Somehow, LLama models stop working normally with tools, if temperature is set to 0

Reply:
It's a bit misleading. `None` is usually interpreted as an unset temperature, not 0.0.
Let's set 0.0 as a default to the temperature argument instead:

```diff
- temperature: float | None = None,
+ temperature: float = 0.0,
```

Reply:
Fixed